addons/mail/static/tests/new/web/chatter_topbar_tests.js

```diff
@@ -261,5 +261,5 @@ QUnit.test("attachment counter transition when attachments become loaded", async
 
     await afterNextRender(() => attachmentPromise.resolve());
     assert.containsNone(target, ".o-mail-chatter-topbar-add-attachments .fa-spin");
-    assert.containsOnce(target, ".o-mail-chatter-topbar-add-attachments:contains(0)");
+    assert.containsNone(target, ".o-mail-chatter-topbar-add-attachments:contains(0)");
```

Comment:
I removed this assertion, already covered by "attachment counter without attachments"